snippets/logging-lokistack-cr-snip.adoc

```diff
@@ -0,0 +1,32 @@
+// Text snippet included in the following assemblies:
+// logging/logging-loki-ocp.adoc
+//
+// Text snippet included in the following modules:
+//
+//
+:_content-type: SNIPPET
+
+[source,yaml]
+----
+apiVersion: loki.grafana.com/v1
+kind: LokiStack
+metadata:
+  name: logging-loki
+  namespace: openshift-logging
+spec:
+  size: 1x.small <1>
+  storage:
+    schemas:
+    - version: v12
+      effectiveDate: "2022-06-01"
+    secret:
+      name: logging-loki-s3 <2>
+      type: s3 <3>
+  storageClassName: "<storage_class_name>" <4>
+  tenants:
+    mode: openshift-logging
+----
+<1> Sizing for Loki follows the format of `N<x>.<size>` where the value `<N>` is number of instances and `<size>` specifies performance capabilities.
```

Comment:
We only offer support for 1x.extra-small, 1x.small and 1x.medium. Do we need to mention it as `N<x>.<size>`?